--- a/Modules/Greetd/GreeterContent.qml
+++ b/Modules/Greetd/GreeterContent.qml
@@ -806,15 +806,7 @@
         asynchronous: false
         cache: true
         visible: source !== ""
-        layer.enabled: true
-
-        layer.effect: MultiEffect {
-            autoPaddingEnabled: false
-            blurEnabled: true
-            blur: Theme.lockScreenBlur
-            blurMax: Theme.lockScreenBlurMax
-            blurMultiplier: 1
-        }
+        layer.enabled: false
 
         Behavior on opacity {
             NumberAnimation {
@@ -850,7 +842,8 @@
         Column {
             id: greeterMainColumn
 
-            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.left: parent.left
+            anchors.leftMargin: 80
             anchors.verticalCenter: parent.verticalCenter
             spacing: Theme.spacingM
             width: Math.min(LockMetrics.passwordRowWidth, parent.width - Theme.spacingXL * 2)
@@ -864,7 +857,7 @@
                 Row {
                     id: clockText
 
-                    anchors.horizontalCenter: parent.horizontalCenter
+                    anchors.left: parent.left
                     anchors.top: parent.top
                     spacing: 0
 
@@ -940,7 +933,7 @@
             StyledText {
                 id: dateText
 
-                anchors.horizontalCenter: parent.horizontalCenter
+                anchors.left: parent.left
                 text: systemClock.date.toLocaleDateString(I18n.locale(), SettingsData.lockDateFormat !== "" ? SettingsData.lockDateFormat : Locale.LongFormat)
                 font.pixelSize: Theme.fontSizeXLarge
                 color: Theme.lockScreenContentColor
