--- a/Modules/Greetd/GreeterContent.qml
+++ b/Modules/Greetd/GreeterContent.qml
@@ -854,15 +854,7 @@
         asynchronous: false
         cache: true
         visible: source !== ""
-        layer.enabled: true
-
-        layer.effect: MultiEffect {
-            autoPaddingEnabled: false
-            blurEnabled: true
-            blur: 0.8
-            blurMax: 32
-            blurMultiplier: 1
-        }
+        layer.enabled: false
 
         Behavior on opacity {
             NumberAnimation {
@@ -897,7 +889,8 @@
         Column {
             id: greeterMainColumn
 
-            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.left: parent.left
+            anchors.leftMargin: 80
             anchors.verticalCenter: parent.verticalCenter
             spacing: Theme.spacingM
             width: 380
@@ -911,7 +904,7 @@
                 Row {
                     id: clockText
 
-                    anchors.horizontalCenter: parent.horizontalCenter
+                    anchors.left: parent.left
                     anchors.top: parent.top
                     spacing: 0
 
@@ -1023,7 +1016,7 @@
             StyledText {
                 id: dateText
 
-                anchors.horizontalCenter: parent.horizontalCenter
+                anchors.left: parent.left
                 text: systemClock.date.toLocaleDateString(I18n.locale(), GreetdSettings.getEffectiveLockDateFormat())
                 font.pixelSize: Theme.fontSizeXLarge
                 color: "white"
