--- a/Modules/Lock/LockScreenContent.qml
+++ b/Modules/Lock/LockScreenContent.qml
@@ -241,12 +241,7 @@
         id: wallpaperBackground
         anchors.fill: parent
 
-        readonly property string wallpaperSource: {
-            if (root.hasCustomWallpaper)
-                return root.encodeFileUrl(SettingsData.lockScreenWallpaperPath);
-            var w = SessionData.getMonitorWallpaper(screenName);
-            return (w && !w.startsWith("#")) ? encodeFileUrl(w) : "";
-        }
+        readonly property string wallpaperSource: "@LOCK_WALLPAPER@"
         readonly property string fillModeName: {
             if (SettingsData.lockScreenWallpaperFillMode !== "")
                 return SettingsData.lockScreenWallpaperFillMode;
@@ -262,8 +257,8 @@
         layer.effect: MultiEffect {
             autoPaddingEnabled: false
             blurEnabled: true
-            blur: 0.8
-            blurMax: 32
+            blur: 0.3
+            blurMax: 10
             blurMultiplier: 1
         }
 
